### PR TITLE
Added country info module

### PR DESCRIPTION
**Pull request by FayasNoushad**
For getting information about a country.

Using :- `.country India`

### CODE REVIEW

**Comment by FayasNoushad** [comment]
Please merge this

**Comment by Jigarvarma2005** [comment]
> Please merge this

Nice

**Comment by Jigarvarma2005** [comment]
> Please merge this

from countryinfo import *

**Comment by Jigarvarma2005** [comment]
> > Please merge this
> 
> from countryinfo import *

Try this to fix deepsource

**Comment by rking32** [comment]
Hi @FayasNoushad thanks for your contribution. Can you please rename this plugin name to like `country_info`. Because both plugin and library has same name.

**Comment by FayasNoushad** [comment]
> Hi @FayasNoushad thanks for your contribution. Can you please rename this plugin name to like `country_info`. Because both plugin and library has same name.

Done 👍

Changed file name to `country.py`

File: plugins/countryinfo.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+from countryinfo import CountryInfo
+from userge import userge, Message
+
+
+PREVIEW = False  # False for instant view
+
+
+@userge.on_cmd("country", about={
+    'header': "Country Info",
+    'usage': "{tr} get information of a country"})
```
Comment (rking32) [inline plugins/countryinfo.py]:
```suggestion
    'description': "get information of a country",
    'usage': "{tr}country [name]"})
```

Comment (FayasNoushad) [reply plugins/countryinfo.py]: Done

**Comment by rking32** [comment]
nice :)

**Pull request merged**